developer-workflow/communication-channels.rst

```diff
@@ -75,10 +56,11 @@ on Libera.Chat_.
 issue tracker is the place to discuss and work on improvements to the CPython
 core development workflow.
 
-A complete list of Python mailing lists can be found at https://mail.python.org/mailman/listinfo.
-Most lists are also mirrored at `GMANE <https://gmane.io/>`_ and can be read and
-posted to in various ways, including via web browsers, NNTP newsreaders, and
-RSS feed readers.
+A complete list of Python mailing lists can be found at
+https://mail.python.org/mailman/listinfo (older lists, using Mailman2) or
+https://mail.python.org/archives/ (newer lists, using Mailman3). Some lists may also
```

Comment:
Actually, these aren't _quite_ the same—the existing [mailman(2) link](https://mail.python.org/mailman/listinfo) points to the listing of list _info_ pages, e.g. [pages like this](https://mail.python.org/mailman/listinfo/python-list) (i.e. , whereas the [mailman3 link](https://mail.python.org/archives/) points to the list of Hyperkitty list _archives_, e.g. [pages like this](https://mail.python.org/archives/list/python-dev@python.org/), while the equivalent of the former for mailman3 is https://mail.python.org/mailman3/, i.e. currently linked by the target ``web gateway`` (AFAIK, there's no direct equivalent for Pipermail, other than, well, [this](https://mail.python.org/pipermail/).

So I suggest perhaps:

```suggestion
A complete list of Python mailing lists can be found at
https://mail.python.org/mailman/listinfo (older lists, using Mailman2) or
is https://mail.python.org/mailman3/ (newer lists, using Mailman3).
Some lists may also
```